lib/ad_service/query/for_audience_creation.ex

```diff
@@ -1,17 +1,13 @@
 defmodule AdService.Query.ForAudienceCreation do
   import Ecto.Query
 
-  def metrics(filters) do
-    {excluded_countries, filters} =
-      filters
-      |> split_countries_and_filters
-
+  def metrics([{:included_countries, included_countries} | filters]) do
     from(
       impression in CodeFund.Schema.Impression,
       where:
         impression.inserted_at >= ^beginning_of_last_month() and
```

Comment:
This might be replaceable with the Property#estimated_monthly_visitors (this prevents us from requiring one month of history for each property)